web/src/beta/components/fields/URLField/index.tsx

```diff
@@ -13,6 +12,8 @@ import { useT } from "@reearth/services/i18n";
 import { useNotification, useWorkspace } from "@reearth/services/state";
 import { styled } from "@reearth/services/theme";
 
+import TextInput from "../common/TextInput";
```

Comment:
We don't replace exists fields using new ui components. We'll:
- 1. implement new ui components like TextInput
- 2. implement new fields like URLField
- 3. replace where the old URLField been used with the new one.